docs/Single-server-VictoriaMetrics.md

```diff
@@ -343,20 +343,59 @@ See also [vmagent](https://docs.victoriametrics.com/vmagent.html), which can be
 
 VictoriaMetrics accepts data from [DataDog agent](https://docs.datadoghq.com/agent/) or [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/) via ["submit metrics" API](https://docs.datadoghq.com/api/latest/metrics/#submit-metrics) at `/datadog/api/v1/series` path.
 
-### Single-node VictoriaMetrics:
+### Sending metrics to VictoriaMetrics
 
-Run DataDog agent with environment variable `DD_DD_URL=http://victoriametrics-host:8428/datadog`. Alternatively, set `dd_url` param at [DataDog agent configuration file](https://docs.datadoghq.com/agent/guide/agent-configuration-files/) to `http://victoriametrics-host:8428/datadog`.
+DataDog agent allows configuring destinations for metrics sending via ENV variable `DD_DD_URL` or via [configuration file](https://docs.datadoghq.com/agent/guide/agent-configuration-files/) in section `dd_url`.
 
-### Cluster version of VictoriaMetrics:
+<p align="center">
+  <img src="Sending_DD_metrics_to_VM.png" width="800">
+</p>
+
+Depending on where you want to send metrics, the URL for VictoriaMetrics will be different:
+1. Run DataDog using ENV variable `DD_DD_URL=http://victoriametrics-host:8428/datadog` for sending metrics to single-node VictoriaMetrics. Alternatively add `dd_url: http://victoriametrics-host:8428/datadog` to DataDog yaml configuration file.
+2. Run DataDog using variable `DD_DD_URL=http://vminsert-host:8480/insert/0/datadog` for sending metrics to the vminsert component of the cluster version of VictoriaMetrics. Alternatively add `dd_url: http://vminsert-host:8480/insert/0/datadog` to DataDog yaml configuration file.
```

Comment:
wdyt about adding datadog endpoints for single/cluster to https://docs.victoriametrics.com/url-examples.html
and just refer them here?

Reply:
Good point
Done. Please check and let me know wdyt.